sdk/python/endpoints/online/model-1/onlinescoring/score_managedidentity.py

```diff
@@ -4,10 +4,31 @@
 import numpy
 import joblib
 import requests
-from pathlib import Path
+from azure.identity import ManagedIdentityCredential
+from azure.storage.blob import BlobClient
 
 
-def get_token():
+def access_blob_storage_sdk():
+    credential = ManagedIdentityCredential()
```

Comment:
Same here, I think this should be
```suggestion
    client_id = os.getenv('UAI_CLIENT_ID')
    credential = ManagedIdentityCredential(client_id=client_id)
```